packages/starlight/__e2e__/fixtures/basics/src/content.config.ts

```diff
@@ -5,9 +5,13 @@ import { glob } from 'astro/loaders';
 
 export const collections = {
 	docs: defineCollection({ loader: docsLoader(), schema: docsSchema() }),
-	// A collection not handled by Starlight.
+	// Collections not handled by Starlight.
 	reviews: defineCollection({
 		loader: glob({ base: './src/content/reviews', pattern: `**/[^_]*.{md,mdx}` }),
 		schema: z.object({ title: z.string() }),
 	}),
+	comments: defineCollection({
+		loader: glob({ base: './src/content/comments', pattern: `**/[^_]*.{md,mdx}` }),
+		schema: z.object({ title: z.string() }),
+	}),
```

Comment:
Should the comments here reflect the Markdown usage?

```suggestion
	// A collection not handled by Starlight.
reviews: defineCollection({
loader: glob({ base: './src/content/reviews', pattern: `**/[^_]*.{md,mdx}` }),
schema: z.object({ title: z.string() }),
}),
	// A collection opted in to Starlight’s Markdown processing.
	comments: defineCollection({
		loader: glob({ base: './src/content/comments', pattern: `**/[^_]*.{md,mdx}` }),
		schema: z.object({ title: z.string() }),
	}),
```